cmd/root.go

```diff
@@ -74,7 +79,7 @@ func Execute() error {
 }
 
 func doPrint(paths []string, fn func(*tfconf.Module) (string, error)) {
-	module, err := tfconf.CreateModule(paths[0])
+	module, err := tfconf.CreateModule(paths[0], outputValuesPath)
```

Comment:
Since we're starting to add more variables to `CreateModule` function I'd suggest we create a struct of its own, and have that as the only argument for `CreateModule(option Option)`. something like, `module.Option` in `tfconf/options.go` which has the following:

- `Path string`
- `OutputValues bool`
- `OutputValuesFile string`

Reply:
I've just pushed #193, you can take a look and let me know. We can proceed with that one first and then you rebase, or we can proceed with this PR (disregarding my comment above) and then I'll rebase the other one. Feel free to let me know which one you prefer.

Reply:
Looks like yours is a lot more complete than what I've done for this portion. I can rebase after you merge :smile_cat:

Reply:
Cool, so I'm gonna merge that and please rebase here and add required properties to the new `Options`.